doc/source/features/online-schema-changes.rst

```diff
@@ -0,0 +1,82 @@
+.. _online-schema-changes:
+
+===================================================
+Online Schema Changes
+===================================================
+
+A failure in a schema change can cause your cluster to drop nodes and lose data. |Percona XtraDB Cluster| supports the following methods for making online schema changes:
```

Comment:
Do we want to start a page describing schema changes by mentioning a failure, without first explaining what an online schema change is, or how and why a failure can occur?

Reply:
Fixed.